src/components/admin/create-tank.vue

```diff
@@ -24,8 +24,7 @@
 <script lang="ts">
 import Vue from 'vue';
 import Cookie from 'js-cookie';
-import { Tank, BrewhopsCookie } from '../../types';
-import { setTimeout } from 'timers';
+import { Tank, BrewhopsCookie } from '@/types/index';
```

Comment:
In the diff above you say `src/types/index` instead of `@/types/index`, are these synonymous? If so, can we be consistent.

Reply:
They are all consistent. `@` is set as an alias to the `src` directory in the `tsconfig.json`. Its a default setting.

Reply:
I think I took care of this. I've made sure these are all consistent.